### DISCUSSION TITLE
Status column customization stopped applying

### DISCUSSION THREAD

**Discussion by patrick-5546** [LazyVim/LazyVim#4758] [Q&A] [answered]
I had my status column configured to show both relative and absolute line numbers in my options.lua file
```lua
vim.opt.statuscolumn = "%s%r %l "
```

However I just noticed that I only see relative numbers now, I'm not sure when this broke. In fact, nothing happens no matter what I change this variable to. Any ideas what might have caused this?

**Comment by dpetka2001** [comment]
```lua
{
  "folke/snacks.nvim",
  opts = {
    statuscolumn = { enabled = false },
  },
}
```

**Reply by folke** [reply, marked as answer]
fyi: this work-around is no longer needed on `main`

**Reply by dpetka2001** [reply]
Oh nice. Just saw the latest changes in the PR for the next release. Thanks for clarifying :smile:
